src/config/config_def.c

```diff
@@ -178,6 +178,11 @@ static const WT_CONFIG_CHECK confchk_WT_CONNECTION_reconfigure[] = {
 	{ NULL, NULL, NULL, NULL, NULL, 0 }
 };
 
+static const WT_CONFIG_CHECK confchk_WT_CONNECTION_rollback_to_stable[] = {
+	{ "timestamp", "string", NULL, NULL, NULL, 0 },
+	{ NULL, NULL, NULL, NULL, NULL, 0 }
+};
```

Comment:
@michaelcahill's alternative is also viable.  My only thought if you go with that API suggestion would be to wonder whether a call where the global `stable_timestamp` isn't set should be an error or a successful no-op.  My suggestion of a more generally named API came about from Eric's API request email coming at the same time I was looking at this.